BattleNetwork/bindings/bnScriptEnvironmentManager.cpp

```diff
@@ -43,6 +43,9 @@
 #include "../bnPlayerPackageManager.h"
 #include "../bnMobPackageManager.h"
 
+template<class T>
+void DefineType( sol::table& battle_namespace, sol::table& overworld_namespace, sol::table& engine_namespace );
```

Comment:
A few commits into your review and I don't see this used anywhere?

Reply:
This file was rolled back into `bnScriptResourceManager.cpp` because it was initially meant to break things up to reduce compile times.
Was beaten to the punch, so removed this and put the code that was moved to this back where it was initially to avoid unnecessary merge conflicts.